api/users.go

```diff
@@ -53,23 +60,44 @@ func getUserMiddleware(w http.ResponseWriter, r *http.Request) {
 		panic(err)
 	}
 
+	editor := context.Get(r, "user").(*db.User)
+	if editor.Admin != true && editor.ID != user.ID {
+		log.Warn(editor.Username + " doesn't permitted for user editing")
```

Comment:
should be `editor.Username + " doesn't permit user editing"` here and in other locations.
Sorry for the pedantry about english sentence construction!

Reply:
Done. That's alright, UI should have clear sentences, without any mess.